### DISCUSSION TITLE
Pull request

### DISCUSSION THREAD

**Discussion by ishitaojha** [community/community#17741] [New to GitHub]
I have forked a repo, and merged my pull request in the forked one , will the changes show in the Original repository as well ?

**Comment by pattacini** [comment]
Nope. Pull requests need to be opened up with the base branch of the upstream repository if you aim to make them show up therein.

**Reply by pattacini** [reply]
Actually, what I wrote above is not fully true in that from the upstream repository one can always get a summary of the commits, including forks, by visiting the `Insights/Network` tab.
